rellic/AST/IRToASTVisitor.cpp

```diff
@@ -899,8 +936,14 @@ void IRToASTVisitor::visitCastInst(llvm::CastInst &inst) {
   if (cast) {
     return;
   }
+
+  auto oper = inst.getOperand(0);
```

Comment:
A cast without an operand? What triggers this?

Reply:
This was here from debugging some issues

Reply:
Fixed